### ISSUE TITLE
Vertical scroll table (not whole page) on small windows

### ISSUE THREAD

**Issue by dchiller** [DDMAL/CantusDB#769]
NewCantus now has a horizontal scroll bar when windows are small enough that all table columns cannot be displayed. However, these tables (eg. on the chant search page) are very "tall" and since vertical scrolling happens at the page level, you have to scroll all the way to the bottom of the page to access the horizontal scroll. 

Top of table, no horizontal scroll: 

![image](https://github.com/DDMAL/CantusDB/assets/11023634/2a2f8353-e7f5-4c06-b31d-8eff28464d16)

Bottom of table, horizontal scroll:

![image](https://github.com/DDMAL/CantusDB/assets/11023634/a0631f16-b624-4302-8aa8-ee3719ea10b3)

This makes the horizontal scroll practically unusable by mouse (it is still usable with arrow keys). Putting the vertical scroll at the table level would make the horizontal scroll visible throughout the table, with the additional benefit that if you scrolled only the table rows, you could keep the header frozen.

**Comment by jacobdgm** [comment]
I like this suggestion (particularly the keeping the header frozen) - thanks!
